packages/eas-cli/src/build/metadata.ts

```diff
@@ -63,6 +63,7 @@ export async function collectMetadataAsync<T extends Platform>(
     customWorkflowName: ctx.customBuildConfigMetadata?.workflowName,
     developmentClient: ctx.developmentClient,
     requiredPackageManager: ctx.requiredPackageManager ?? undefined,
+    selectedImage: ctx.buildProfile.image,
   };
   return sanitizeMetadata(metadata);
```

Comment:
We also need to update `sanitizeMetadata` and bump here, don't we?

Reply:
that's right, thanks

Reply:
https://github.com/expo/eas-build/pull/285